packages/frontend-core/src/components/grid/cells/HeaderCell.svelte

```diff
@@ -24,13 +25,19 @@
     definition,
     datasource,
     schema,
+    focusedCellId,
+    filter,
   } = getContext("grid")
 
+  const searchableTypes = ["string", "options", "number", "array", "longform"]
```

Comment:
Can we use the types from `FieldType`?

Reply:
I definitely will now that they're typed! That's great to have 👌